src/lib/template/TemplateFetch.ts

```diff
@@ -121,9 +121,13 @@ class TemplateFetchURL {
         await this.gitClone(urlParts.url, cacheDirectory, urlParts.b);
       }
     } catch (e) {
-      console.error('Could not clone or pull the given git repository!');
+      console.error('Could not clone or pull the given git repository, clearing and cloning from fresh.');
       this.cleanSingleCacheDir(cacheDirectory);
-      throw e;
+      try {
+        await this.gitClone(urlParts.url, cacheDirectory, urlParts.b);
+      } catch (e) {
```

Comment:
I dont think you need to catch to throw again, but works

Reply:
i'll roll it out now but i will try to check  that later

thanks for checking